docs/csharp/language-reference/statements/iteration-statements.md

```diff
@@ -83,7 +83,7 @@ The `foreach` statement executes a statement or a block of statements for each e
 
 The `foreach` statement isn't limited to those types. You can use it with an instance of any type that satisfies the following conditions:
 
-- A type has the public parameterless `GetEnumerator` method. The `GetEnumerator` method can be a type's [extension method](../../programming-guide/classes-and-structs/extension-methods.md).
+- A type has the public parameterless `GetEnumerator` method. The `GetEnumerator` method can be a type's [extension member](../../programming-guide/classes-and-structs/extension-methods.md).
```

Comment:
I prefer the original "method" in this case. Or is the term "extension method" going away?